apps/site/src/views/Resources/sections/ApiResources/ApiResources.tsx

```diff
@@ -27,11 +27,11 @@ async function ApiResources() {
 				{resources.map(
 					({ _id, title, description, link, logo, stickyNoteColor }) => (
 						<div className={styles.column + " col"} key={_id}>
-							<ApiGroup
+							<ResourceCard
 								title={title}
 								description={<PortableText value={description} />}
 								stickerSrc={imageUrlBuilder(client).image(logo).url()}
-								tagLink={link}
+								links={[{ text: "API Reference", link: link }]}
```

Comment:
Thought [non-blocking]: Unsure if link labels should originate from CMS or be predetermined. Worth further discussion later.